content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -429,40 +365,33 @@ pipeline {
 }
 // Script //
 ----
-<1> An `agent` must be set at the top level of the pipeline. This will fail if agent is set as `agent none`.
-<2> When using `returnStdout` a trailing whitespace will be appended to the returned string. Use `.trim()` to remove this.
+<1> An `agent` must be set at the top level of the pipeline.
+This will fail if agent is set as `agent none`.
+<2> When using `returnStdout` a trailing whitespace will be appended to the returned string.
+Use `.trim()` to remove this.
 
 === Handling credentials
 
 Credentials
-link:../../using/using-credentials#configuring-credentials[configured in
-Jenkins] can be handled in Pipelines for immediate use. Read more about using
-credentials in Jenkins on the link:../../using/using-credentials[Using
-credentials] page.
+link:../../using/using-credentials#configuring-credentials[configured in Jenkins] can be handled in Pipelines for immediate use.
+Read more about using credentials in Jenkins on the link:../../using/using-credentials[Using credentials] page.
 
 .The correct way to handle credentials in Jenkins
 video::yfjtMIDgmfs[youtube,width=800,height=420]
 
 
 ==== For secret text, usernames and passwords, and secret files
 
-Jenkins' declarative Pipeline syntax has the `credentials()` helper method (used
-within the <<syntax#environment,`environment`>> directive) which supports
-<<#secret-text,secret text>>, <<#usernames-and-passwords,username and
-password>>, as well as <<#secret-files,secret file>> credentials. If you want to
-handle other types of credentials, refer to the <<#for-other-credential-types,
-For other credential types>> section (below).
+Jenkins' declarative Pipeline syntax has the `credentials()` helper method (used within the <<syntax#environment,`environment`>> directive) which supports <<#secret-text,secret text>>, <<#usernames-and-passwords,username and password>>, as well as <<#secret-files,secret file>> credentials.
+If you want to handle other types of credentials, refer to the <<#for-other-credential-types, For other credential types>> section (below).
```

```suggestion
If you want to handle other types of credentials, refer to the <<#for-other-credential-types, For other credential types>> section.
```
Suggestion to remove `below` per [microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/below)